src/components/entity/__internal__/entity.spec.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+
+import Entity from './entity.component';
+import EntityHeader from './entity-header.component';
+import EntityContent from './entity-content.component';
+import EntityContentDivider from './entity-content-divider.component';
+import {
+  StyledEntityHeaderContainer, StyledEntityHeader, StyledEntityContentDivider
+} from './entity.style';
+
+import { MenuButton } from '../../action-popover/action-popover.style';
+import StyledIcon from '../../icon/icon.style';
+
+import { baseTheme } from '../../../style/themes';
+import { assertStyleMatch } from '../../../__spec_helper__/test-utils';
```

Comment:
Why there are blank lines? I didn't see the convention in other files. I think we should to remove them to be consistant to other componets/files

Reply:
See my earlier comment.